src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -291,15 +291,13 @@ class HuiEntitiesCard extends LitElement implements LovelaceCard {
 
   private renderEntity(entityConf: LovelaceRowConfig): TemplateResult {
     let config = entityConf as EntityConfig;
-    if (this._config?.state_color) {
-      config = { state_color: true, ...config } as EntityConfig;
-    }
-    if (this._config?.secondary_info) {
-      config = {
-        secondary_info: this._config!.secondary_info,
-        ...config,
-      } as EntityConfig;
-    }
+
+    config = {
+      state_color: this._config!.state_color ?? undefined,
+      secondary_info: this._config!.secondary_info ?? undefined,
```

Comment:
Are you sure `this._config` is always defined?

Reply:
Yes, this is called in `render` and there is a check on it at the top